demos/check_for_unreads.js

```diff
@@ -6,12 +6,16 @@ async function main() {
 
   try {
     await bot.init({username: process.env.KB_USERNAME, paperkey: process.env.KB_PAPERKEY, verbose: false})
-    const res = await bot.chatList({
+    const res = await bot.chat.list({
       showErrors: true,
-      unreadOnly: true,
+      unreadOnly: false,
```

Comment:
I think this should be `false`, since we filter the unreads and compare them to the total